src/docs/patterns/internationalization.mdx

```diff
@@ -13,42 +16,83 @@ import Image4 from '../../app/assets/design-patterns/internationalization/web-in
 
-### Showing Language Options
+## Translating the Text
 
-When showing a list of languages, you can decide whether you want to translate the language names or not (e.g., "English" vs. "Inglés"). There are pros and cons to each approach, so consider what makes the most sense for your application. Providing other cues (such as an icon) to help users find the language selection can help.
+To localize your application content, you should create a string resource file or object that contains the translations for any display text used in your application. You will not want to have any text hard-coded in your application's UI. To switch your application to another language, you simply toggle the file/object used to populate the UI to the appropriate translation.
 
-## Bi-directionality/ Right-to-Left (RTL) Support
+## Converting Units, Currencies, and Number Formats
 
-Some languages, such as Arabic and Hebrew, are read from right to left. Applications supporting these languages will need to mirror the UI so the natural flow is from right to left. For example, navigation drawers should be displayed on the right side of the screen, icons should be to the right of the text in buttons, and any icons with directionality should be mirrored.
+While this usually goes with the locale settings, certain users might have different preferences, and there can be variations even within the same locale. You should provide the user with default settings when they first load a locale, but allow the users change their preferences, and remember it the next time they login with that locale.
+
+## Checking for Spacing
+
+Different languages/writing systems may require different vertical and horizontal space. To ensure flexibility, avoid using fixed dimensions for necessary UI components such as buttons, labels, and badges. Also, consider setting different default display font sizes for languages that may have smaller glyphs.
+
+> **NOTE:** PX Blue recommends using [Noto Sans](https://www.google.com/get/noto/#sans-lgc) for all non-western languages. You can read more on our [Typography](/style/typography) page.
+
+## Bi-directionality/Right-to-Left (RTL) Support
 
 <MaterialDesignDescription
     title={'Bidirectionality'}
     description={'Learn about mirroring layout, mirroring elements, and localization.'}
     url={'https://material.io/design/usability/bidirectionality.html'}
 />
 
-## Technical Implementation
+Some languages, such as Arabic and Hebrew, are read from right to left. Applications supporting these languages will need to mirror the UI so the natural flow is from right to left. For example, navigation drawers should be displayed on the right side of the screen, icons should be to the right of the text in buttons, and any icons with directionality should be mirrored.
```

Comment:
Should we also mention the flipping of certain icons (and link to the wonderful deep link where Material explains which to flip)?

Reply:
That wonderful link is right on the top, between line 56 and line 60.

Reply:
Yeah, but the part about which icons to flip is 4 miles down on that Material.io page. It's probably fine, I was just wondering if it might be nice to deep link specifically to the icon part in this context.

Reply:
Added an additional inline deeplink